src/main/java/com/shippo/model/Batch.java

```diff
@@ -0,0 +1,357 @@
+package com.shippo.model;
+
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLDecoder;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+import com.shippo.exception.APIConnectionException;
+import com.shippo.exception.APIException;
+import com.shippo.exception.AuthenticationException;
+import com.shippo.exception.InvalidRequestException;
+import com.shippo.net.APIResource;
+
+/*
+ * Represents ../batches endpoint documented at https://goshippo.com/docs/reference#batches
+ */
+public final class Batch extends APIResource {
+
+	@SerializedName("object_id")
+	private String id;
+
+	@SerializedName("object_owner")
+	private String owner;
+
+	public static enum BatchStatus {
+		VALIDATING, VALID, INVALID, PURCHASING, PURCHASED
+	}
+
+	@SerializedName("object_status")
+	private BatchStatus status;
+
+	@SerializedName("object_created")
+	private Date created;
+
+	@SerializedName("object_updated")
+	private Date updated;
+	private String metadata;
+	private String defaultCarrierAccount;
+	private String defaultServiceLevelToken;
```

Comment:
Good catch. Thanks! GSON stores `a_b` property in `aB` but it can be changed using `SerializedName` annotation. I've added that annotation for all multi-word properties just to be explicit.